web_app/backend/tests/test_main.py

```
@@ -0,0 +1,81 @@
+class MockResponse:
+    def __init__(self, success=True, content="Test response"):
+        self.success = success
+        self.content = content
+        self.overall_confidence = 0.95
+        self.models_used = ["gpt-4"]
+        
+        class MetaData:
+            total_execution_time = 1.2
+            execution_path = []
```

Ruff (0.15.1)

[warning] Mutable default value for class attribute

(RUF012)